src/nft/components/icons.tsx

```diff
@@ -4,7 +4,7 @@ import styled, { useTheme } from 'styled-components/macro'
 import { themeVars, vars } from '../css/sprinkles.css'
 
 // ESLint reports `fill` is missing, whereas it exists on an SVGProps type
-type SVGProps = React.SVGProps<SVGSVGElement> & { fill?: string }
+type SVGProps = React.SVGProps<SVGSVGElement> & { fill?: string; height?: string | number; width?: string | number }
```

Comment:
i'm surprised this was necessary tbh. if you feel it's good we can investigate if there's a better option here